rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -642,7 +498,7 @@ private void setHoverNodeLabel() {
             this.hoverLabel.setContents(str);
         }
 
-        public void setSelectedTile(boolean selected) {
+        void setSelectedTile(boolean selected) {
             if (selected) {
                 this.getElement().getStyle().setBackgroundColor("#7caaf7");
                 this.getElement().getStyle().setBorderColor("#d9e4f6");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "#d9e4f6" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)